examples/envoy/docker/spire-server/conf/spire-server.conf

```diff
@@ -6,7 +6,8 @@ server {
 	data_dir = "/opt/spire/data/server"
 	log_level = "DEBUG"
 	log_file = "/opt/spire/server.log"
-	default_svid_ttl = "1h"
+	default_x509_svid_ttl = "1h"
```

Comment:
That's true, technically it's not needed since the values are the same. I was thinking perhaps it is worthwhile to keep this configuration though since it is a very common one for users to think about. Including it brings more attention to it for new users, since these examples are usually viewed by people who are brand new to SPIRE and learning about how to configure/deploy it.

There's also an argument that it adds unnecessary complexity to the config, so I could go either way on this. WDYT @azdagron ?

Reply:
I can go either way as well but I've found that in many cases, folks just copy the config without thinking about it (as you can see with the noop node resolver). I'd rather have the example be as bare bones as possible. If there are a set of configurables that folks need to consider deeply, we should probably centralize that in some sort of "here's what you should think about when deploying spire" kind of doc.

Reply:
Sounds good. One less config for us to maintain as well.